app/src/services/unwrap_token.ts

```diff
@@ -14,11 +14,9 @@ class UnwrapTokenService {
     const symbolLowerCase = symbol.toLowerCase()
     switch (symbolLowerCase) {
       case 'wxdai':
-        return wxDaiabi
       case 'wspoa':
-        return wsPOAabi
       case 'weth':
-        return wETHabi
+        return wrappedAsset
```

Comment:
Doesn't this mean we're not returning anything for wxdai and wsPOAabi?

Reply:
well those that were deleted are return values not the cases for evaluation. So it will be fine...

Reply:
Why not just remove the switch and return `wrappedAsset` since they're all the same ABI?

Reply:
Oh I guess this is the same thing effectively(?)

Reply:
Hmm yeah I guess maybe I can do it like this? But my only concern was with the default option where it returns empty object? Do you think that can cause issues?
![Screenshot 2021-08-17 at 22 29 55](https://user-images.githubusercontent.com/33226956/129796142-0f5b74a5-cbd2-48f7-9654-6e9fc7205c44.png)

Reply:
This was is before?
![Screenshot 2021-08-17 at 22 31 12](https://user-images.githubusercontent.com/33226956/129796311-874eb3f0-1051-40eb-8119-da1fb398dde2.png)

Reply:
I just pushed it and tested it. You were right! Its way better like this got ridd of lots of code....